src/db/configuration/getConfiguration.ts

```diff
@@ -178,6 +178,8 @@ export const getConfiguration = async (): Promise<Config> => {
         minedTxListenerCronSchedule: "*/5 * * * * *",
         maxTxsToUpdate: 50,
         retryTxListenerCronSchedule: "*/30 * * * * *",
+        indexerListenerCronSchedule: "*/5 * * * * *",
+        maxBlocksToIndex: 5,
```

Comment:
Do we want the users to toggle these values? if Yes, we need to add these new configs under the `configuration` end-points, and allow user to change this.

Reply:
maybe this was not clear,

we need to add `/configuration/indexer` GET & POST and include the above properties.